db/models/chains.js

```diff
@@ -3,21 +3,22 @@ const {
   Model
 } = require('sequelize');
 module.exports = (sequelize, DataTypes) => {
-  class paybuttons_addresses extends Model {
+  class chains extends Model {
     /**
      * Helper method for defining associations.
      * This method is not a part of Sequelize lifecycle.
      * The `models/index` file will call this method automatically.
      */
     static associate(models) {
-        paybuttons_addresses.paybuttons = paybuttons_addresses.belongsTo(models.paybuttons, { as: 'paybutton' })
+        chains.addresses = chains.hasMany(models.paybutton_addresses, { as: 'addresses' });
     }
   }
-  paybuttons_addresses.init({
-    address: DataTypes.STRING
+  chains.init({
+    slug: DataTypes.STRING,
+    title: DataTypes.STRING
   }, {
     sequelize,
-    modelName: 'paybuttons_addresses',
+    modelName: 'chains',
   });
-  return paybuttons_addresses;
+  return chains;
 };
```

Comment:
Don't rename files only to re-add the old file. It makes finding the history of changes annoying to find. Just add the new file

Reply:
I didn't rename them, I actually deleted them and created new ones, and in different commits, to try to avoid that. I don't understand why the diff is like that

Reply:
I deleted the migrations file on [this](https://github.com/PayButton/paybutton-server/pull/66/commits/bd1f6ab44f42f492dc39b9b940dcd84cb0dccb23) commit and created the new migrations on [this one](https://github.com/PayButton/paybutton-server/pull/66/commits/70a2c99bb13558e0d4b4853f148612aadb2dbb11).

This is a model file, so what I did doesn't apply to it, nonetheless, this problem of interpreting the deletion + creation as a rename is also happening on the aforementioned migration files, so I think it wouldn't have helped to do the same in the model files.

Reply:
Ok, we can ignore this for now. Next time instead of deleting, try just using `git mv oldFileName newFileName`